python/ql/src/semmle/python/Exprs.qll

```diff
@@ -691,6 +691,26 @@ class StrConst extends Str_, ImmutableLiteral {
 
     override Object getLiteralObject() { none() }
 
+    /* Add some utility methods missing on `string` */
```

Comment:
I don't immediately see where these missing utility methods are actually needed. Perhaps we should hold off on adding them until such a need arises?

Reply:
Yes, probably best to add them to `string` itself at some point.